src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -866,7 +866,7 @@ private void DynamoReadyExtensionHandler(ReadyParams readyParams, IEnumerable<IE
                 }
                 catch (Exception ex)
                 {
-                    Logger.Log(ex.Message);
+                    Logger.Log("Failed to call Ready() in extension: " + ext.Name + " " + ex.Message);
```

Comment:
Are we going to add this to string resources? Also, I think the message could be made more user friendly to be displayed in the Dynamo console.

Reply:
Hi @aparajit-pratap I think this would not need localization but I can add it to resource string for sure. Also from my testing, logging for Dynamo console does not work for extension but only view extension.